samples/Mvc.TagHelperSample.Web/Views/Home/Create.cshtml

```diff
@@ -0,0 +1,94 @@
+﻿
+@using TagHelperSample.Web.Models
+@model User
+
+@{
+    Html.Html5DateRenderingMode = Html5DateRenderingMode.Rfc3339;
+}
+
+<h2>Create</h2>
+
+@* antiforgery is on by default *@
+@* <form/> tag helper will special-case only elements with an "asp-action" or "asp-antiforgery" attribute. *@
```

Comment:
Unless something seems _really_ important, bare this file back to the HTML. Most of the Razor comments are design notes from a meeting a long time ago. Not worthwhile outside the MVC repo.